kdump/include/runtest.sh

```diff
@@ -511,7 +506,82 @@ AppendConfig()
         shift
     done
 
-    rstrnt-report-log -l "${KDUMP_CONFIG}"
+    RhtsSubmit "${KDUMP_CONFIG}"
```

Comment:
I thought we were trying to move away from the `Rhts*` function and towards using restraint?

Reply:
If that is the case, put all rhts- cmd in a wrapper could make it easier to move all rhts-* cmd away in the future. Then what we need to do is to change the call inside RhtsSubmit, instead updating rhts- all over the place.

Reply:
@emmawu  thanks for the PR! Yes we have moved away from RHTS, by converting all the CKI tests to use Restraint compatible commands and removing the RHTS libraries sourced in the code. Would you mind just renaming the function to RstrntSubmit instead to avoid confusion ? Or feel free to file a separate PR. Thanks!

Reply:
Sure. Updated.